pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -86,6 +92,10 @@ const (
 	// latestFileName is the name of a file in the collection which contains the
 	// path of the most recently taken full backup in the backup collection.
 	latestFileName = "LATEST"
+
+	// latestHistoryDirectory is the directory where all 22.1 and beyond
+	// LATEST files will be stored as we no longer want to overwrite it.
+	latestHistoryDirectory = "latest-history"
```

Comment:
nit: how do you feel about just `latest`? the files are anyways going to be something other than `LATEST` very soon. Happy for you to ignore/change this in a later diff where you actually implement the no overwrite.